src/pages/Subscriptions/SubscriptionsMonthlyGood.vue

```diff
@@ -333,6 +326,12 @@ export default {
 		});
 	},
 	computed: {
+		selectedGroupDescriptor() {
+			// eslint-disable-next-line max-len
+			const selectedCategory = this.lendingCategories.find(category => category.value === this.category) || {};
+			// Sanitize and set initial form values.
+			return selectedCategory ? selectedCategory.shortName : 'all borrowers';
```

Comment:
@eddieferrer Careful with this one. I think there is an additional condition in the legacy page that just skips setting this value (and saving this value) if the category in the lenders account doesn't match one of the provided enums. This is setup in the `_serializeMGCategoryOptions` method of AutoDepositView.js and accounts for lenders that have specialized criteria from outside the normal flow.

Reply:
We have a `Preserve existing settings` option for those lenders with an empty value ''.